tests/sentry/snuba/test_discover.py

```diff
@@ -88,11 +189,12 @@ def test_auto_fields_simple_fields(self):
         assert data[0]["user.email"] == "bruce@example.com"
         assert data[0]["release"] == "first-release"
 
-        assert len(result["meta"]) == 4
+        assert len(result["meta"]) == 9
```

Comment:
🤔 so meta doubles when there's an aggregation included? (ie. user.email, release, id, project.id) show up twice? Not sure why yet. Happens on master too though.

Reply:
This is likely because the fields are in both selected_fields and group by.

Reply:
Hrm, I'm going to strip this length check out of this test then since its actually validating the wrong thing.